docs/architecture.md

```diff
@@ -0,0 +1,94 @@
+---
+# Ensure this is first.
+order: -100
+---
+# Architecture
+
+The Altis Cloud architecture is a highly-available, resilient, autoscaling cloud system built to handle any level of traffic. Altis Cloud has been designed from the ground up to be flexible, scalable, and performant, while minimising cost.
+
+Each environment tier includes as standard:
+
+* Global CDN, powered by Amazon CloudFront
+* Load balancer
+* Automatically-scaling application servers (PHP)
+* Database servers (MySQL)
+* Object caching servers (Redis)
+* Search servers (Elasticsearch)
+* Scheduled jobs servers
+* Asset storage & microservices
+
+The application servers run your Altis-based application, and automatically scale to meet demand. This allows your site to respond dynamically to huge traffic spikes while remaining fast, without incurring any additional cost.
+
+Altis manages the deployment, maintenance, and management of these services, providing you with a comprehensive web application platform. The Altis team manages the underlying virtual servers for you, including monitoring and right-sizing.
+
+
+## Overview
+
+Running a web application like Altis requires running multiple infrastructure components. Altis uses dedicated, specialized services for each component, rather than handling each service on one machine. This provides better performance characteristics by ensuring services have the best resources for their need, and allows each service to scale independently.
+
+Every client and every environment is fully segregated from each other using logical controls at the network level, backed by AWS security controls.
+
+Altis Cloud is designed to be highly-available, with redundant servers split across multiple availability zones, which are physically-independent data centres. Each set of availability zones are located in the [same physical region](./origin-locations.md). This ensures resiliency and redundancy of your data and application.
+
+(Note that full HA is not included with all environment tiers; consult your plan information. High availability is also only available on production environments.)
+
+![Architecture diagram](assets/architecture.svg)
```

Comment:
FYI each customer does not have a VPC, which this diagram seems to suggest.

Reply:
It's a semi-intentional fudge, not really relevant (and we're moving towards that anyway)